packages/utils/package.json

```diff
@@ -45,7 +45,9 @@
     "js-yaml": "^3.13.1",
     "winston": "^3.3.3",
     "winston-daily-rotate-file": "^4.5.5",
-    "winston-transport": "^4.3.0"
+    "winston-transport": "^4.3.0",
+    "@types/lockfile": "^1.0.1",
+    "lockfile": "^1.0.4"
```

Comment:
lodestar-utils is a package meant for platform agnostic code, and a lock files requires a filesystem. Could you look into placing this code in a package that is meant only for NodeJS?

Reply:
gotcha! Actually moved it here because I wanted some re-usability :/ But I'll remove it from `lodestar-utils` in a bit. For my own understanding, the packages that are supposed to be node agnostic are:

api, beacon-state-transition, config, fork-choice, params, types, utils and validator.

Will that be accurate?

Reply:
Yes!